app/README.md

```diff
@@ -1,28 +1,229 @@
-# Vue Skeleton Application
+# Common Service Get Token
 
-This node.js skeleton app hosts the Vue Skeleton frontend. It implements a minimal endpoint to allow for Keycloak authentication.
+Common Service Get Token (GETOK) will automate the process for Development teams to get setup to use the common services published on [https://apistore.nrs.gov.bc.ca/store/](https://apistore.nrs.gov.bc.ca/store/).
```

Comment:
This isn't accurate anymore (https://apistore.nrs.gov.bc.ca/store/ is the WebADE secured ws02 gateway)
Maybe see what Matt wants here?

Reply:
I've updated the app and frontend readmes to align with the top-level blurbs.